client/src/components/common/selectRole/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const SelectRole = ({ label, options }) => {
+  return (
+    <div className="selectRole">
```

Comment:
This is a common component make the class name better 
what I mean that the class name should not be relate to role